### PR TITLE
docs: fix blog-related translation issues

### PR DESCRIPTION
**Pull request by EstelleRao**
Changes:
1. Rewrote some sentences and paragraphs to make the language more natural and fluent.
2. Included two more keywords, **OSPP** and **student**, in the title, slug, description, and summary, to improve SEO to some extent.
3. Corrected some major issues caused by machine translation. For example, changed **Summer of Programming** to **the Summer 2020 of Open Source Promotion Plan** and **Tributary Technology** to **API7.ai**.
4. Fixed some typos.

### CODE REVIEW

File: website/blog/2021/06/03/experience-share-from-Apache-APISIX-committer.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
 ---
-title: Experience sharing from Apache APISIX committer - Interview with Summer of Programming
-slug: 2021/06/03/experience-share-from-apache-apisix-committer
+title: First-Hand Experience with Apache APISIX Shared by Student Participants in OSPP 2020
+slug: /blog/2021/06/03/experience-from-ospp-2020-student-participants-and-apache-apisix-committers
```
Comment (yzeng25) [inline website/blog/2021/06/03/experience-share-from-Apache-APISIX-committer.md]:
```suggestion
slug: /blog/2021/06/03/first-hand-experience-with-apache-apisix-shared-by-student-participants-in-ossp-2020
```

Comment (EstelleRao) [reply website/blog/2021/06/03/experience-share-from-Apache-APISIX-committer.md]: fixed
thx

File: website/blog/2021/06/03/firsthand-experience-with-apache-apisix-shared-by-student-participants-in-ospp-2020.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+---
+title: Firsthand Experience with Apache APISIX Shared by Student Participants in OSPP 2020
+slug: /blog/2021/06/03/firsthand-experience-with-apache-apisix-from-ospp-2020-students
+author: "赵若妃"
+authorURL: "https://github.com/Serendipity96"
+authorImageURL: "https://avatars.githubusercontent.com/u/23514812?v=4"
+keywords:
+- APISIX
+- Open Source
```
Comment (SylviaBABY) [inline website/blog/2021/06/03/firsthand-experience-with-apache-apisix-shared-by-student-participants-in-ospp-2020.md]:
```suggestion
keywords:
- APISIX
- Open Source
- OSPP
- Open Source Promotion Plan
- Experience
```

File: website/blog/2021/06/03/firsthand-experience-with-apache-apisix-shared-by-student-participants-in-ospp-2020.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
+---
+title: Firsthand Experience with Apache APISIX Shared by Student Participants in OSPP 2020
+slug: /blog/2021/06/03/firsthand-experience-with-apache-apisix-from-ospp-2020-students
+author: "赵若妃"
+authorURL: "https://github.com/Serendipity96"
+authorImageURL: "https://avatars.githubusercontent.com/u/23514812?v=4"
+keywords:
```
Comment (guitu168) [inline website/blog/2021/06/03/firsthand-experience-with-apache-apisix-shared-by-student-participants-in-ospp-2020.md:7]: We can also add an "API Gateway"

**Comment by takumi913** [comment]
:smiley:  Welcome to the new world. @EstelleRao

File: website/blog/2021/06/03/firsthand-experience-with-apache-apisix-shared-by-student-participants-in-ospp-2020.md (hunk 2 above)
Comment (EstelleRao) [reply website/blog/2021/06/03/firsthand-experience-with-apache-apisix-shared-by-student-participants-in-ospp-2020.md]: Good idea!

File: website/blog/2021/06/03/firsthand-experience-with-apache-apisix-shared-by-student-participants-in-ospp-2020.md (hunk 3 above)
Comment (EstelleRao) [reply website/blog/2021/06/03/firsthand-experience-with-apache-apisix-shared-by-student-participants-in-ospp-2020.md:7]: Yes! I have added the keyword.

**Review by SylviaBABY — APPROVED** [review]

**Review by guitu168 — APPROVED** [review]

**Review by yzeng25 — APPROVED** [review]

**Pull request merged**